environments/values-usdfdev-prompt-processing.yaml

```diff
@@ -0,0 +1,13 @@
+fqdn: k8s.slac.stanford.edu/usdf-prompt-processing-dev
```

Comment:
The other code does need to be a FQDN, not a URL. I think this environment is currently disabling all of the applications for which this would matter (although I think Argo CD's ingress may break?), but I don't think we want to support this syntax and try to teach other things how to handle it going forward. Can SLAC assign usdf-prompt-processing-dev.slac.stanford.edu or the like to this environment?

Reply:
Thanks!  Adjusted to usdf-prompt-processing-dev.slac.stanford.edu